tests/tests/swfs/avm2/cryptscore/test.toml

```diff
@@ -0,0 +1,7 @@
+# This file tests that our Math.random() implementation works
```

Comment:
The only thing that makes me hesitant from LGTM'ing this test is the possibility you mentioned on Discord, that it might randomly fail CI. Though I guess we can disable the test if it happens for the first time...?

Reply:
I believe we seed our rng when running tests, so this shouldn't be an issue